train.py

```diff
@@ -876,6 +888,16 @@ def restore_parts(path, model):
             hparams.parse_json(f.read())
     # Override hyper parameters
     hparams.parse(args["--hparams"])
+    
+    # Preventing Windows specific error such as MemoryError 
+    # Also reduces the occurrence of THAllocator.c 0x05 error in Widows build of PyTorch
+    if platform.system() == "Windows":
+        print("Windows Detected - num_workers set to 1")
+        hparams.set_hparam('num_workers',1)
+
+    # Now, print the finalized hparams.
+    print(hparams_debug_string())
```

Comment:
A really minor point though, `print(hparams_debug_string())` is called twice. I will fix it after merging.